packages/common/src/HardhatConfig.ts

```diff
@@ -147,6 +147,12 @@ export function getHardhatConfig(
         gasPrice: 30000000000,
       },
       mumbai: { chainId: 80001, url: getNodeUrl("polygon-mumbai", true, 80001), accounts: { mnemonic } },
+      amoy: {
+        chainId: 80002,
+        url: getNodeUrl("polygon-amoy", true, 80002),
+        accounts: { mnemonic },
+        gasPrice: 10000000000,
```

Comment:
For some reason automatic gas price detection did not work, so did hardcode to 10Gwei